Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha2/index.md

```diff
@@ -1,18 +1,22 @@
 ---
 versionFrom: 7.0.0
+versionTo: 9.0.0
 ---
 
 # reCAPTCHA V2
 
-![reCAPTCHA v2](images/recaptcha2.png)
-
 You need to configure your site keys adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
```

Comment:
It is the same with ReChapta: https://our.umbraco.com/Documentation/Add-ons/UmbracoForms/Developer/Configuration/index-v9#recaptcha-v2-field-type-configuration :)